### ISSUE TITLE
hi,can the project run on cpu

### ISSUE THREAD

**Issue by tigerdhl** [cyrusbehr/YOLOv8-TensorRT-CPP#59]
Thank u for share your project and there are any plan to modify the project can be run on cpu?

**Comment by cyrusbehr** [comment]
No, that somewhat defeats the purpose. TensorRT is designed for running on GPU. If you want to run on CPU, look into an onnxruntime implementation.
